e2etests/boot_tests.bzl

```diff
@@ -10,3 +10,17 @@ def launch_cvd_boot_test(name, branch, target):
             "no-sandbox",
         ],
     )
+
+def cvd_load_boot_test(name, env_file, size = "medium"):
```

Comment:
do we need boot in the name? why not `cvd_load_test`?

Reply:
Probably not, but that naming scheme is in the other tests. If you want to see it changed I'll do it in a different PR.

The reason for the "boot" in the name is legacy: Cuttlefish is used to run CTS, so when you only want to make sure it boots it's usually called "boot test".